### PR TITLE
Add ability to filter metrics we want to export

### PR DESCRIPTION
**Pull request by migibert**
Hi

### CODE REVIEW

**Comment by migibert** [comment]
I signed it!

File: contrib/dropwizard/src/test/java/io/opencensus/contrib/dropwizard/DropWizardMetricsTest.java [hunk 1]
```diff
@@ -321,4 +323,24 @@ public void collect_Timer() throws InterruptedException {
...
+    dropWizardMetrics = new DropWizardMetrics(Collections.singletonList(metricRegistry), filter);
+    Timer timerTestRequests = metricRegistry.timer("test_requests");
+    metricRegistry.timer("requests");
+
+    Collection<Metric> metrics = dropWizardMetrics.getMetrics();
+    assertThat(metrics).hasSize(1);
+
+    Metric value = metrics.iterator().next();
+    assertThat(value.getMetricDescriptor().getName()).isEqualTo("codahale_test_requests_timer");
+
```
Comment (bogdandrutu) [inline contrib/dropwizard/src/test/java/io/opencensus/contrib/dropwizard/DropWizardMetricsTest.java]: Remove unnecessary empty line here.

File: contrib/dropwizard/src/main/java/io/opencensus/contrib/dropwizard/DropWizardMetrics.java [hunk 2]
```diff
@@ -68,9 +70,23 @@
...
+   * @param metricFilter a filter to choose which metric to export
+   * @since 0.19
+   */
+  public DropWizardMetrics(
+      List<com.codahale.metrics.MetricRegistry> metricRegistryList, MetricFilter metricFilter) {
     Utils.checkListElementNotNull(
         Utils.checkNotNull(metricRegistryList, "metricRegistryList"), "metricRegistry");
+    Utils.checkNotNull(metricFilter, "metricFilter");
     this.metricRegistryList = metricRegistryList;
+    this.metricFilter = metricFilter;
```
Comment (bogdandrutu) [inline contrib/dropwizard/src/main/java/io/opencensus/contrib/dropwizard/DropWizardMetrics.java]: Combine line 87 and 89:
this.metricFilter = Utils.checkNotNull(metricFilter, "metricFilter");

**Comment by bogdandrutu** [comment]
Please rebase to resolve the conflict in the CHANGELOG.

**Pull request merged**